src/styles/card.scss

```
@@ -0,0 +1,92 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  box-sizing: border-box;
+  background-color: #fff;
+  border: solid 1px #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: #060b35;
+    margin: 40px 16px 0;
+  }
+
+  &__code {
+    line-height: 14px;
+    font-weight: 400;
+    font-size: 10px;
+    color: #616070;
+    margin: 4px 16px 0;
+  }
+
+  &__reviews {
+    display: flex;
+    box-sizing: border-box;
+    flex-direction: row;
+    justify-content: space-between;
+    margin: 16px 16px 0;
+    align-items: center;
+    height: 16px;
+  }
+
+  &__reviews-value {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: #060b35;
+  }
+
+  &__price {
+    height: 18px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin: 24px 16px 0;
+    align-items: center;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: #616070;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: #060b35;
+  }
```

Use scss nesting, fix it everywhere
```suggestion
  &__price {
    height: 18px;
    display: flex;
    flex-direction: row;
    justify-content: space-between;
    margin: 24px 16px 0;
    align-items: center;
    
    &-text {
      // write styles here
    }
    
    &-value {
      // write styles here
    }
  }
```